ROADMAP.md

```diff
@@ -0,0 +1,191 @@
+
+#### I have all the documentation I need as a Maintainer
+
+- public specifications
+- public discussions
+- community gatherings
+
+#### Efficient Project Management and Project Visibility
+
+- We have a single maintainer team for review,
+- We have a clear label and triaging process,
+- We have a reliable & transparent contribution process (protected branches, etc),
+- We have precise project management tooling (re-ignite ZenHub?)
+
+### 5. Testground provides the networking tooling required to test complex Distributed / Decentralized Applications
+
+**Why:** This is Testground's main feature.
+
+#### Milestone 1: ExplicitNetwork Simulation Support
+
+There is a clear matrix of what features can be used with which Testground runner.
+
+- MTU issues, networking simulation, etc.
+- This will be a Matrix of feature x runner x precision (transport level, application level, etc.).
+
+#### Milestone 2: Testground provides support for important networks topologies
+
+- Access to public networks - [issue 1472](https://github.com/testground/testground/issues/1472)
+- NAT simulation - [issue 1299](https://github.com/testground/testground/issues/1299)
+- Complex topologies - [issue 1354](https://github.com/testground/testground/issues/1354)
+
+#### Milestone 3: Testground provides a way to run tests over real networks
+
+- Remote runners feature
+
+### 6. Testground provides the tooling to make test maintenance & finding issues simple
+
+**Why:**
+
+- composition files specification and improvements -
+- Logging improvements - [Epic 1355](https://github.com/testground/testground/issues/1355)
+- tcpdump'ing features - [Issue #1384](https://github.com/testground/testground/issues/1384)
+
+### 7. Testground covers every /essential/combination of languages and libraries required by its users
+
+**Why:** Drive adoptions outside of Protocol Labs.
+
+#### We support the primary languages needed by our users
+
+- Provide a simple matrix about which languages and builders are supported and how
+  - example: go support = great, nodejs support = deprecated, python support = non existent
+- Provide a way to raise requests for more language support
+- Provide an example  + SDK for "most" languages using `docker:generic`
+  - `rust`,  `browser`, `nim`, `python`,
+- Provide an official `docker:xxxx` builder for "most" languages (`docker:rust`, `docker:browser`)
+  - This will require deep knowledge of the packaging systems and how to do library rewrites, etc. See the `lipb2p/test-plans` rust support for this.
+
+#### We support the tooling needed to run interop tests in CI
+
+- Custom setup & composition generation scripts
+
+#### We Support SDK Implementers
+
+- Record every known SDK and its level of maintenance in an official "awesome-testground" page
+  - example: nim-sdk.
+- Provide instructions and a path for SDK implementers
+
+## Later
+
+### Testground is used at ProtocolLabs to measure performances
+
+### Testground security is top-notch
```

Comment:
This seems to be a bit far down the list :P

Reply:
Got it.  I think this is more of a distraction.  I would remove it.